src/Controller/UI/ReportResearchGroupDatasetStatusController.php

```diff
@@ -77,52 +78,152 @@ public function defaultAction(Request $request, EntityHandler $entityHandler, in
         );
     }
 
+    /**
+     * Research group dataset status report for dataset monitoring.
+     *
+     * @param Request       $request         Message information for this Request.
+     * @param EntityHandler $entityHandler   The entity handler.
+     *
+     * @Route(
+     *     "/report-researchgroup/dataset-monitoring",
+     *     name="pelagos_app_ui_reportresearchgroupdatasetstatus_datasetmonitoringreport",
+     *     methods={"GET"}
+     *     )
+     *
+     * @return Response|StreamedResponse A Symfony Response instance.
+     */
+    public function datasetMonitoringReportAction(Request $request, EntityHandler $entityHandler)
+    {
+        // Checks authorization of users
+        if (!$this->isGranted('ROLE_DATA_REPOSITORY_MANAGER')) {
```

Comment:
I think the report should be accessible for everyone.

Reply:
I guess this is the admin version?

Reply:
Yes, the post action is called by the button on dataset-monitoring which is accessible by everyone